content/include/table_driven_parallel.md

```diff
@@ -0,0 +1,40 @@
+    func TestParseCadenceArgument(t *testing.T) {
+        tests := []struct {
```

Comment:
I assume it's by intent to use a real-world example of a function.. But I wonder if having this large test setup detracts from the point being made about the loop variable.. The point of the whole thing is the single line no. 28.

Reply:
True that. I'll change it and use the example from the testing package itself, they have a nice concise example there :)